shpc/main/registry/remote.py

```diff
@@ -202,19 +176,19 @@ def _update_clone_cache(self):
         """
         logger.warning(
             "Remote %s is not deploying a Registry API, falling back to clone."
-            % self.source
+            % self.url
         )
-        tmpdir = self.clone()
-        for dirname, module in self.iter_modules():
+        tmplocal = self.clone()
```

Comment:
It looks like we are switching between tmplocal and self commands - e.g., the self will hit `self._cache` and tmplocal is a direct pointer to it. This can be easy for the developer to forget (e.g., why did we do self.cleanup when we were working with tmplocal? I don't have preference for which handle is used, but could we be consistent to use the same one in the various spots of this function? :thinking: